codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/model/AWSDeprecatedShapeRemover.kt

```diff
@@ -28,7 +28,7 @@ class AWSDeprecatedShapeRemover : SwiftIntegration {
             Predicate<Shape> {
                 val since = it.getTrait<DeprecatedTrait>()?.since?.orElse(null) ?: return@Predicate false
                 val deprecatedDate = since.toLocalDate() ?: return@Predicate false.also {
-                    println("Failed to parse `since` field $since as a date, skipping removal of deprecated shape $it")
+                    println("Could not parse `since` field \"$since\" as a date, skipping removal of deprecated shape $it")
```

Comment:
Changed this language because the word "failed" makes it difficult so search logs for test failures.